openbb_platform/core/pyproject.toml

```diff
@@ -9,17 +9,18 @@ packages = [{ include = "openbb_core" }]
 
 [tool.poetry.dependencies]
 python = "^3.9"
-uvicorn = "^0.24"
-websockets = "^12.0"
+uvicorn = "^0.32.0"
+websockets = "^13.0"
+starlette = ">=0.40.0"
 pandas = ">=1.5.3"
 html5lib = "^1.1"
-fastapi = "^0.111.0"
+fastapi = "^0.115.0"
 uuid7 = "^0.1.0"
 posthog = "^3.3.1"
 python-multipart = "^0.0.7"
 pydantic = "^2.5.1"
 requests = "^2.32.1"
-importlib-metadata = "^6.8.0"
```

Comment:
Can you elaborate please why this change is required?

Reply:
There is no need to pin this to a lower version that is obsolete from last year. Current version is 8.5